pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -254,11 +255,26 @@ func validateServiceInstanceUpdate(instance *sc.ServiceInstance) field.ErrorList
 }
 
 // internalValidateServiceInstanceUpdateAllowed ensures there is not a
-// pending update on-going with the spec of the instance before allowing an update
-// to the spec to go through.
-func internalValidateServiceInstanceUpdateAllowed(new *sc.ServiceInstance, old *sc.ServiceInstance) field.ErrorList {
+// pending update on-going with the spec of the instance before allowing an
+// update to the spec to go through unless its the same user who made the
+// original update.
+func internalValidateServiceInstanceUpdateAllowed(ctx genericapirequest.Context, new *sc.ServiceInstance, old *sc.ServiceInstance) field.ErrorList {
 	errors := field.ErrorList{}
-	if old.Generation != new.Generation && old.Status.CurrentOperation != "" {
+
+	oldUID := ""
+	newUID := ""
+
+	if old.Spec.UserInfo != nil {
+		oldUID = old.Spec.UserInfo.UID
+	}
+
+	if ctx != nil {
+		if user, _ := genericapirequest.UserFrom(ctx); user != nil {
+			newUID = user.GetUID()
```

Comment:
We should be getting the UID of the new user directly from the ServiceInstance rather than from the context
 The new user is set in the ServiceInstance as part of preparing the update, which happens before validation.

Reply:
that would certainly be easier! :-)  trying it now....